docs/logging/docs/02-01-logging.md

```diff
@@ -2,8 +2,23 @@
 title: Architecture
 ---
 
-This document outlines the logging architecture of Kyma. It highlights information sources from which Logspout extracts logs to feed to OK Log.
+This document provides an overview of the logging architecture in Kyma. It describes the information sources from which promtail extract logs to feed to Loki.
 
-Logspout is deployed as a stateless Daemonset and shares `/var/run/docker.sock` from the node. It then feeds the logs to OK Log through the ingest API.
+## Agent (Promtail)
+Promtail is the agent responsible for collecting reliable metadata, consistent with the time series, or metrics metadata. To achieve this, the agent uses the same service discovery and label relabelling libraries as Prometheus. The agent is wrapped in a daemon that discovers targets, produces metadata labels, and tails log files to produce a stream of logs buffered on the client side and then sent to the service.
 
-OK Log is deployed as a Statefulset. It is capable for storing the logs for 7 days, which is a configurable property. Read more on OK Log architectural decisions [here](https://github.com/oklog/oklog/tree/master/doc/arch).
+## Life of a write request
+The server-side components on the write path will mirror the [Cortex](https://github.com/cortexproject/cortex) architecture.
+* Writes will first hit the Distributor, which is responsible for distributing and replacing the writes to the ingesters. Loki use the Cortex consistent hash ring and distribute writes based on a hash of the entire metadata.
+* Next writes will hit a 'log ingester' which batches up writes for the same stream in memory in to 'log chunks'. When chunks reach a predefined size or age, periodically flushed to the Cortex chunk store.
+* The Cortex chunk store will be updated to reduce copying of chunk data on the read and write path and add support for writing chunks of google cloud storage.
+
+#### Log Chunks
+A log chunk consists of all logs for a given metadata (e.g. labels), collected over a certain time period. Log chunks support append, seek, and stream operations on read requests.
+
+#### Life of a Query Request
+As chunks are larger than Prometheus/Cortex chunks (Cortex chunks are max 1KB in size), it is not possible to load and decompress them in their entirety. 
```

Comment:
We can't say Prometheus or Cortex because Cortex is a storage engine which prometheus use to store chunks like loki. What I want to highlight here was "Prometheus Cortex chunks are max 1K but loki Cortex chunks can be larger"

Reply:
I changed it to Prometheus-Cortex .